<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>101</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_all">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>431</height>
      </rect>
     </property>
     <attribute name="label">
      <string>总览</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>95</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="b_allp">
         <property name="text">
          <string>全部照片</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_content">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>431</height>
      </rect>
     </property>
     <attribute name="label">
      <string>按内容浏览</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>95</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="b_things">
         <property name="text">
          <string>事物</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_face">
         <property name="text">
          <string>人脸</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_grain">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>431</height>
      </rect>
     </property>
     <attribute name="label">
      <string>按纹理浏览</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="b_g">
         <property name="text">
          <string>点击浏览</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>9</y>
      <width>671</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>671</width>
        <height>521</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>669</width>
         <height>519</height>
        </rect>
       </property>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>19</y>
          <width>591</width>
          <height>491</height>
         </rect>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>591</width>
           <height>491</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2"/>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>611</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="B_aminals">
         <property name="text">
          <string>飞禽走兽</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="B_transport">
         <property name="text">
          <string>交通市政</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="B_sthings">
         <property name="text">
          <string>随身物品</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>食物家居</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="B_sports">
         <property name="text">
          <string>运动相关</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>69</y>
        <width>591</width>
        <height>441</height>
       </rect>
      </property>
      <widget class="QWidget" name="page_t">
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>19</y>
          <width>571</width>
          <height>391</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>569</width>
           <height>389</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_a"/>
      <widget class="QWidget" name="page_b"/>
      <widget class="QWidget" name="page_s"/>
      <widget class="QWidget" name="page_f"/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>210</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>这是页面三</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>这是页面四</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>stackedWidget</zorder>
   <zorder>toolBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionadd_folder"/>
    <addaction name="actionadd_one"/>
    <addaction name="actionadd_pl"/>
    <addaction name="actiondel"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actioncreater"/>
    <addaction name="actionhelp"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionadd_folder">
   <property name="text">
    <string>添加文件夹</string>
   </property>
  </action>
  <action name="actioncreater">
   <property name="text">
    <string>作者信息</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="actionadd_one">
   <property name="text">
    <string>添加单张照片</string>
   </property>
  </action>
  <action name="actionadd_pl">
   <property name="text">
    <string>添加多张照片</string>
   </property>
  </action>
  <action name="actiondel">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
